### PR TITLE
Dev/145059 delta exchange set display accurate size

### PR DESCRIPTION
**Pull request by anuj104124**
Hi,

This PR contains the unit test changes.

Please review the same.

Thanks

### CODE REVIEW

**Review by richardAhz — APPROVED** [review]
LGTM

File: Tests/UnitTests/ess-list-encs.component.spec.ts [hunk 1]
```diff
@@ -516,6 +516,22 @@ describe('EssListEncsComponent', () => {
     };
     expect(essInfoErrorMessageService.infoErrMessage).toStrictEqual(warnObj);
   });
+
+  test('getEstimatedTotalSize should return esimatedTotalSize when selectedEncList length is 0', () => {
```
Comment (gaurav15233) [inline Tests/UnitTests/ess-list-encs.component.spec.ts:520]: Please correct the spelling mistake esimatedTotalSize to estimatedTotalSize

File: Tests/UnitTests/ess-list-encs.component.spec.ts [hunk 2]
```diff
@@ -516,6 +516,22 @@ describe('EssListEncsComponent', () => {
...
+    component.selectedEncList = [];
+    component.getEstimatedTotalSize();
+    expect(component.selectedEncList.length).toBe(0);
+    expect(component.getEstimatedTotalSize()).toBe('0 MB');
+  });
+
+  test('getEstimatedTotalSize should return esimatedTotalSize when selectedEncList length is greater than 0', () => {
```
Comment (gaurav15233) [inline Tests/UnitTests/ess-list-encs.component.spec.ts:527]: Please correct the spelling mistake esimatedTotalSize to estimatedTotalSize

**Review by gaurav15233 — COMMENTED** [review]
Suggested minor changes

**Review by jagannathsankpal — APPROVED** [review]
Code changes looks good to me

File: Tests/UnitTests/ess-list-encs.component.spec.ts (hunk 1 above)
Comment (anuj104124) [reply Tests/UnitTests/ess-list-encs.component.spec.ts]: Resolved

File: Tests/UnitTests/ess-list-encs.component.spec.ts (hunk 2 above)
Comment (anuj104124) [reply Tests/UnitTests/ess-list-encs.component.spec.ts]: Resolved

**Review by gaurav15233 — APPROVED** [review]
LGTM

**Pull request merged**